packages/react/time-series/src/types.ts

```diff
@@ -1,3 +1,13 @@
+import type { ScaleOptionsByType } from 'chart.js'
+
+export type DeepPartial<T> = T extends object
+  ? {
+      [P in keyof T]?: DeepPartial<T[P]>
+    }
+  : T
```

Comment:
This is a work around. We need to find out how correctly type this. If chart.js change their definition we would have TS errors

Reply:
They are not exporting this type directly, we would need to import this one

```ts
import type { DeepPartial } from 'chart.js/dist/types/utils'
```

Maybe it's not the ideal but I think it could be better than defining our own `DeepPartial` type, wdyt?

Reply:
definitely!